Mcol 294

LinuxJedi opened the pull request:
Switch to using the operating system jemalloc if one is found at compile time.

If not then jemalloc will not be used.

Note that this also fixes an unrelated namespace issue which was breaking compilation on some platforms.

davidwbt commented on build/nightly-mini-tester.sh:
```diff
@@ -65,12 +65,6 @@ find ${prefix}/Calpont -type f | xargs chmod +r
 
 mkdir -p ${prefix}/Calpont/data1/systemFiles/dbrm
```
While not related to the pull request - i see references to Calpont directories, i wonder if this is even exercised since the installs should be in columnstore now?  Similarly in src-build below.

LinuxJedi commented:
I don't think the shell file you highlighted is even used any more. I just updated it due to references to things I changed. There are several scripts which I think are probably dead now.

davidhilldallas commented:
Correct, we still need to cleanout some of the older calpont scripts.. they arent being built..
David Hill 


Sent via the Samsung Galaxy S7, an AT&T 4G LTE smartphone
-------- Original message --------From: Andrew Hutchings <notifications@github.com> Date: 9/26/16  10:07 AM  (GMT-06:00) To: mariadb-corporation/mariadb-columnstore-engine <mariadb-columnstore-engine@noreply.github.com> Subject: Re: [mariadb-corporation/mariadb-columnstore-engine] Mcol 294 (#18) 
I don't think the shell file you highlighted is even used any more. I just updated it due to references to things I changed. There are several scripts which I think are probably dead now.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/mariadb-corporation/mariadb-columnstore-engine","title":"mariadb-corporation/mariadb-columnstore-engine","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/mariadb-corporation/mariadb-columnstore-engine"}},"updates":{"snippets":[{"icon":"PERSON","message":"@LinuxJedi in #18: I don't think the shell file you highlighted is even used any more. I just updated it due to references to things I changed. There are several scripts which I think are probably dead now."}],"action":{"name":"View Pull Request","url":"https://github.com/mariadb-corporation/mariadb-columnstore-engine/pull/18#issuecomment-249597987"}}}

benthompson15 commented:
Several files in the build directory are dead / leftover from the previously used autotools configure/build. Probably need to be cleaned up eventually.
